src/models/Winners.js

```diff
@@ -0,0 +1,17 @@
+class Winners {
+  static findWinners(carList) {
+    let maxDistance = 0;
+    let winners = [];
+    carList.forEach(car => {
```

Comment:
forEach 로 해주신 방법도 너무 좋은 것 같습니다! 추가적으로 말씀그리면 이런식으로 distance값 중에서 가장 큰 값을 찾고, 그 값과 일치하는 자동차들을 필터링해 winners 배열로 만드는 것도 방법입니다! (이렇게 하면 매번 배열을 초기화 할 필요가 없다는 장점이 있어용!)
```suggestion
   const maxDistance = Math.max(...carList.map(car => car.distance));

   const winners = carList
       .filter(car => car.distance === maxDistance)
       .map(car => car.name);
```